### ISSUE TITLE
Reason: INTERNAL SERVER ERROR returned from prod when asking for advise

### ISSUE THREAD

**Issue by pacospace** [thoth-station/thoth-application#1125]
**Describe the bug**
Reason: INTERNAL SERVER ERROR returned from prod when asking for advise

**To Reproduce**
1. Ask advise from thoth

**Expected behavior**
adviser workflow is correctly scheduled

**Screenshots**
```
error locking dependencies using Thoth: (500)
Reason: INTERNAL SERVER ERROR
HTTP response headers: HTTPHeaderDict({'Server': 'gunicorn', 'Date': 'Mon, 29 Mar 2021 11:03:06 GMT', 'Content-Type': 'application/json', 'Content-Length': '177', 'X-Thoth-Version': '0.22.1', 'X-User-API-Service-Version': '0.22.1+messaging.0.11.2.storages.0.39.2.common.0.27.0.python.0.15.6', 'Access-Control-Allow-Origin': '*', 'Set-Cookie': '99770cb82864be05282857f803e02327=deaa8f13dfb3a7c16de0db759f79d09b; path=/; HttpOnly; Secure; SameSite=None'})
HTTP response body: {"details":{"datetime":"2021-03-29T11:03:06.366748","type":"InternalServerError"},"error":"Internal server error occurred, please contact administrator with provided details."}
```

**Additional context**

**Comment by pacospace** [comment]
This is still relevant!

**Comment by pacospace** [comment]
> The issue seems related to messaging version error.
> 
> ```
> {"name": "thoth.user_api.openapi_server", "levelname": "ERROR", "module": "app", "lineno": 1891, "funcname": "log_exception", "created": 1618304388.9621856, "asctime": "2021-04-13 08:59:48,962", "msecs": 962.1856212615967, "relative_created": 42871068.86553764, "process": 207, "message": "Exception on /api/v1/advise/python [POST]\nTraceback (most recent call last):\n  File \"/opt/app-root/lib64/python3.8/site-packages/flask/app.py\", line 2447, in wsgi_app\n    response = self.full_dispatch_request()\n  File \"/opt/app-root/lib64/python3.8/site-packages/flask/app.py\", line 1952, in full_dispatch_request\n    rv = self.handle_user_exception(e)\n  File \"/opt/app-root/lib64/python3.8/site-packages/flask_cors/extension.py\", line 165, in wrapped_function\n    return cors_after_request(app.make_response(f(*args, **kwargs)))\n  File \"/opt/app-root/lib64/python3.8/site-packages/flask/app.py\", line 1821, in handle_user_exception\n    reraise(exc_type, exc_value, tb)\n  File \"/opt/app-root/lib64/python3.8/site-packages/flask/_compat.py\", line 39, in reraise\n    raise value\n  File \"/opt/app-root/lib64/python3.8/site-packages/flask/app.py\", line 1950, in full_dispatch_request\n    rv = self.dispatch_request()\n  File \"/opt/app-root/lib64/python3.8/site-packages/flask/app.py\", line 1936, in dispatch_request\n    return self.view_functions[rule.endpoint](**req.view_args)\n  File \"/opt/app-root/lib64/python3.8/site-packages/connexion/decorators/decorator.py\", line 48, in wrapper\n    response = function(request)\n  File \"/opt/app-root/lib64/python3.8/site-packages/connexion/decorators/uri_parsing.py\", line 144, in wrapper\n    response = function(request)\n  File \"/opt/app-root/lib64/python3.8/site-packages/connexion/decorators/validation.py\", line 184, in wrapper\n    response = function(request)\n  File \"/opt/app-root/lib64/python3.8/site-packages/connexion/decorators/validation.py\", line 384, in wrapper\n    return function(request)\n  File \"/opt/app-root/lib64/python3.8/site-packages/connexion/decorators/parameter.py\", line 121, in wrapper\n    return function(**kwargs)\n  File \"/opt/app-root/src/thoth/user_api/api_v1.py\", line 410, in post_advise_python\n    response, status = _send_schedule_message(message, AdviserTriggerMessage)\n  File \"/opt/app-root/src/thoth/user_api/api_v1.py\", line 959, in _send_schedule_message\n    message = message_type.MessageContents(**message_contents)\nTypeError: __init__() got an unexpected keyword argument 'kebechet_metadata'"}
> ```

This error should be solved for `user-api >= 0.23.0`

**Comment by harshad16** [comment]
The production version is v0.24.3
the advisers run applying 
we are monitoring other issues if any other occurs.
closing this issue.
please feel free to open another one if seen any new issue.
